{"event": "verbose", "uuid": "6b20e317-18e6-47c1-b13f-eaa594b03ea9", "counter": 126, "stdout": "\u001b[0;31mERROR! Invalid extra vars data supplied. '@/Users/devops/data/project/python/pgoops/upload/iac/test6/env/extravars' could not be made into a dictionary\u001b[0m", "start_line": 125, "end_line": 126, "runner_ident": "f479754a-f324-4e35-9edb-13d537ceef7d", "created": "2022-04-21T17:25:44.777580"}